File: aurora-common/src/main/java/xyz/xcye/common/vo/MessageLogVO.java
package xyz.xcye.common.vo;

import lombok.Data;
import xyz.xcye.common.valid.Delete;
import xyz.xcye.common.valid.Insert;
import xyz.xcye.common.valid.Update;
import xyz.xcye.common.valid.validator.ValidateString;

/**
 * @author qsyyke
 */

@Data
public class MessageLogVO {
    /**
     * 唯一uid
     */
    private Long uid;

    /**
     * 生产者生产的消息
     */
    private String message;

    /**
     * 交换机
     */
    private String exchange;

    /**
     * 队列
     */
    private String queue;

    /**
     * 绑定路由key
     */
    private String routingKey;

    /**
     * 重试次数
     */
    private Integer tryCount;

    /**
     * 消息消费的状态，true
     */
    private Boolean consumeStatus;

    /**
     * 创建时间
     */
    private String createTime;

    /**
     * 修改时间
     */
    private String updateTime;

    /**
     * 交换机类型
     */
    private String exchangeType;
}

File: src/theme/vo/message/MessageLogVo.ts
interface MessageLogVo {
  /**
   * 唯一uid
   */
  uid?: string | null;

  /**
   * 投递的消息
   */
  message?: string | null;

  /**
   * 交换机名称
   */
  exchange?: string | null;

  /**
   * 队列名称
   */
  queue?: string | null;

  /**
   * 绑定路由key
   */
  routingKey?: string | null;

  /**
   * 重试次数
   */
  tryCount?: number | null;

  /**
   * 1: 表示消费成功 0：表示消费失败
   */
  consumeStatus?: boolean | null;

  /**
   *
   */
  createTime?: string | null;

  /**
   *
   */
  updateTime?: string | null;

  /**
   * 交换机类型
   */
  exchangeType?: string | null;

  /**
   * 确认状态 1：应答了
   */
  ackStatus?: boolean | null;

  /**
   * 如果发生错误，则错误消息是什么
   */
  errorMessage?: string | null;
}

export { MessageLogVo };
